### ISSUE TITLE
Yes, the existing paragraphs are really hard to parse for difference between the roles. Something like this would be more clear:

### ISSUE THREAD

**Issue by Capitalxtends** [googleads/google-ads-php#909]
Yes, the existing paragraphs are really hard to parse for difference between the roles. Something like this would be more clear:

| Role | Member | Manager | Owner | Billing Manager |
| ----------- | ----------- | - | - | - |
| Own/maintain specific projects | ✅ |  ✅ |  ✅ |  ❌ | 
| Create new projects | ❌ |  ✅ |  ✅ |  ❌ | 
| Create/manage teams | ❌ |  ✅ |  ✅ |  ❌ | 
| Invite/manage organization members | ❌ |  ❌ |  ✅ |  ❌ | 
| Delete projects | ❌ |  ❌ |  ✅ |  ❌ | 
| Delete the organization | ❌ |  ❌ |  ✅ |  ❌ | 
| Manage billing | ✅ |  ✅ |  ✅ |  ✅ | 

(It's unclear to me who exactly has the 'manage billing' permission as well, that last line might be wrong).

> Seems the minimum viable resolution here would be to change the last sentence above to say "Can only manage billing for the organization."?

That makes it more explicit for the last one, but the challenges parsing the differences between the other roles remains.

_Originally posted by @di in https://github.com/pypi/warehouse/issues/12431#issuecomment-1301053492_

**Comment by fiboknacky** [comment]
Looks like this is posted in a wrong place, so I'm closing this.
